pkg/validators/am0001_default_channel.go

```diff
@@ -15,7 +15,7 @@ var AM0001 = types.Validator{
 	Code:        "AM0001",
 	Name:        "default_channel",
 	Description: "Ensure defaultChannel is present in list of channels",
-	Runner:      validateDefaultChannel,
+	Runner:      types.ValidateFunc(validateDefaultChannel),
```

Comment:
compiler will complain if `validateDefaultChannel` does not implement `ValidatorRunner` interface. Is extra cast necessary?

Reply:
See comment on `ValidateFunc`